pkg/virt-operator/install-strategy/strategy.go

```diff
@@ -78,6 +79,9 @@ type InstallStrategy struct {
 	deployments                     []*appsv1.Deployment
 	daemonSets                      []*appsv1.DaemonSet
 	validatingWebhookConfigurations []*v1beta1.ValidatingWebhookConfiguration
+	mutatingWebhookConfigurations   []*v1beta1.MutatingWebhookConfiguration
+	apiServices                     []*v1beta12.APIService
+	certificateSecrets              []*corev1.Secret
```

Comment:
do you have an idea on how we can remove secrets from the install strategy?

Reply:
Regarding to managing them, I think it is good to finally have them there. I don't save the content of the secrets. The content is dynamically checked and filled in in create.go of the operator.

Reply:
any reason to validate secret content is black during install strategy generation?

Reply:
Could you rephrase that?

Reply:
@davidvossel could you explain what you meant with your sentence?